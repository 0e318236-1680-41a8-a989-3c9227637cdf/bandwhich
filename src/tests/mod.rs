pub mod fakes;
pub mod cases;
