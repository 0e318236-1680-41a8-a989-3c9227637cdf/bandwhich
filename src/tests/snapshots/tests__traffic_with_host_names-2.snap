---
source: src/tests/mod.rs
expression: "&terminal_draw_events_mirror[2]"
---
                     29    42                                                                                                                                                                 
                                                                                                                                                                                              
                                                                                                                                                                                              
                                                                                                                                                                                              
         three.thre  three.three:1337 (tcp)          5                     12    26             5                                                                         12    26            
         one.one.on  one:12345 (tcp)                 1                      7     6             1                                                                          7     6            
                                                                                                                                                                                              
                                                                                                                                                                                              
                                                                                                                                                                                              
                                                                                                                                                                                              
                                                                                                                                                                                              
                                                                                                                                                                                              
                                                                                                                                                                                              
                                                                                                                                                                                              
                                                                                                                                                                                              
                                                                                                                                                                                              
                                                                                                                                                                                              
                                                                                                                                                                                              
                                                                                                                                                                                              
                                                                                                                                                                                              
                                                                                                                                                                                              
                                                                                                                                                                                              
                                                                                                                                                                                              
                                                                                                                                                                                              
                                                                                                                                                                                              
                                                                                                                                                                                              
                                                                                                                                                                                              
                                                                                                                                                                                              
                                                                                                three.thre  three.three                                                   12    26            
                                                                                                one.one.on  one                                                            7     6            
                                                                                                                                                                                              
                                                                                                                                                                                              
                                                                                                                                                                                              
                                                                                                                                                                                              
                                                                                                                                                                                              
                                                                                                                                                                                              
                                                                                                                                                                                              
                                                                                                                                                                                              
                                                                                                                                                                                              
                                                                                                                                                                                              
                                                                                                                                                                                              
                                                                                                                                                                                              
                                                                                                                                                                                              
                                                                                                                                                                                              
                                                                                                                                                                                              
                                                                                                                                                                                              
                                                                                                                                                                                              
                                                                                                                                                                                              
                                                                                                                                                                                              
                                                                                                                                                                                              

